README.md

```diff
@@ -29,15 +29,24 @@ This site relies heavily on node and npm.
 1. Make sure you are using at least node v.5+ and latest npm; 
 if not install [nvm](https://github.com/creationix/nvm) to get node going on your machine.
 
-1. Install npm `gulp` package *globally*: `npm install -g gulp`
+1. Install global npm packages by running `./scripts/before-install.sh`
```

Comment:
There's only one other mention of the scripts in the readme (the one for resetting the environment). 

I guess it's fine, but will keep an eye out to see if new authors or users have having trouble.